### DISCUSSION TITLE
namespace in conjunction with room does not work

### DISCUSSION THREAD

**Discussion by d1ksim** [miguelgrinberg/python-socketio#1144] [Q&A]
Greetings to all, I noticed such a problem if out of context I use 
`await sio.emit('new_message_response', data={'test':'tester'}, namespace='/chat', room='c50c277a-b3a8-49e8-84ae-ed2497ed6655')`

Then nothing happens, but if you remove the room, then sending happens, the room exists and there are people in it.

**Comment by miguelgrinberg** [comment]
Can I ask you to not write bugs on this project unless you know for a fact that you are experiencing a bug and are prepared to provide all the requested information? If all you have is a question, please use the discussions forum.

With regards to your question, I suggest you enable Socket.IO logs to see  what is the activity when you issue the emit.

**Comment by maybephilipp** [comment]
I use python SIO with namespaces and rooms without any problem. In your case, I think, most likely your SIO Client didn't use namespace in connection statement.
